release/ray_release/byod/byod.Dockerfile

```diff
@@ -18,4 +18,5 @@ sudo apt-get update -y \
 EOF
 
 COPY "$PIP_REQUIREMENTS" .
+RUN "$HOME"/anaconda3/bin/pip install --no-cache-dir https://ray-ci-deps-wheels.s3.us-west-2.amazonaws.com/AutoROM.accept_rom_license-0.5.4-py3-none-any.whl
```

Comment:
how big is this thing? do we need this for most of the tests?

Reply:
It's 500kb, only a few test needs this. This should have gone into requirements_byod but it doesn't not have a pip version